app.js

```diff
@@ -0,0 +1,91 @@
+const operations = require('./dboperations');
+const express = require('express');
+const cors = require('cors');
+const app = express();
+
+const appointment = require('./Appointment');
+const donor = require('./Donor');
+
+app.use(express.urlencoded());
+app.use(express.json());
+app.use(cors());
+
+
+
+const port = process.env.port || 3000;
+app.listen(port);
+console.log("Location API is running at port: " + port);
+
+app.use((request, response, next) =>{
+    console.log('middleware');
+    next();
+})
+
+
+//OPERATIONS
+
+//Location
+app.get('/locations', (request, response) => {
+    operations.getLocations().then(result => {
+        response.json(result[0]);
+    })
+});
+
+//Appointments
+app.get('/getappointments/:id', (request, response) =>{
+    operations.getAppointments(request.params.id).then(result =>{
+        response.json(result);
+    })
+})
+
+app.delete('/deleteappointment/:id', (request, response) =>{
+    operations.deleteAppointment(request.params.id).then(result =>{
+        response.status(201).json(result);
+    })
+})
+
+app.post('/registerappointment', (request, response) =>{
+    let appointment = {...request.body}
```

Comment:
Doing a spread on `request.body` is a bit scary, as any body properties sent via HTTP request will be copied.

E.g: I can populate multiple dummy properties, and they'll all be included with this object.

Would be better if you map them one by one, specifying only what you need. E.g:

```javascript
const appointment = {
    name: request.body.name,
    ....and so on
}
```